setup.py

```diff
@@ -33,8 +33,12 @@
         "intervaltree",
         "simplejson",
         "textx",
-        "prjxray@git+git://github.com/SymbiFlow/prjxray.git#egg=prjxray",
-        "fasm@git+git://github.com/SymbiFlow/fasm.git#egg=fasm",
+        "prjxray",
+        "fasm",
+    ],
+    dependency_links=[
+        'https://github.com/Symbiflow/fasm/tarball/master'
+        'https://github.com/Symbiflow/prjxray/tarball/master'
```

Comment:
These should be identical to the ones in `requirements.txt`, could you update them to use the same locations?

Reply:
Fixed